internal/service/federatedsettingsidentityprovider/data_source_federated_settings_identity_provider_test.go

```diff
@@ -36,6 +36,35 @@ func TestAccFedDSFederatedSettingsIdentityProvider_basic(t *testing.T) {
 	})
 }
 
+func TestAccFedDSFederatedSettingsIdentityProvider_oidcBasic(t *testing.T) {
+	acc.SkipTestExtCred(t)
```

Comment:
would it be possible easy to have a migration test even with acc.SkipTestExtCred(t) or it's hard to do?

Reply:
I would say that it's not straight forward even with acc.SkipTestExtCred(t) but I am open to discussing it. Maybe as part of enabling acceptance tests in the CI we could create migration tests?

Reply:
This could be good test to add to ensure no breaking changes are encountered. It can import a resource with previous version and then update to the latest and expect an empty plan.

Reply:
+1 with the test. Perhaps no need to block the release (and this PR) but it's a good one to have, we should actually aim at always have a test like this.

Reply:
I tried to add the migration test with the help of @lantoli and it seems that there is an issue when running a migration test using `ExternalProvider` and `ImportState` in the acceptance test step at the same time. We will look again after release.